kyuubi-server/src/test/scala/org/apache/kyuubi/operation/KyuubiOperationKerberosAndPlainAuthSuite.scala

```diff
@@ -65,8 +68,29 @@ class KyuubiOperationKerberosAndPlainAuthSuite extends WithKyuubiServer with Ker
         classOf[UserDefineAuthenticationProviderImpl].getCanonicalName)
   }
 
-  test("test with KERBEROS authentication") {
-    val conn = DriverManager.getConnection(jdbcUrlWithConf(kerberosJdbcUrl), user, "")
+  test("test with KERBEROS TGT cache authentication") {
+    assume(Utils.isCommandAvailable("kinit"))
+    // run kinit to generate tgt cache
+    val commands = Seq("kinit", "-kt", testKeytab, testPrincipal)
+    val kinitProc = new ProcessBuilder(commands: _*).inheritIO()
+    kinitProc.environment().put("KRB5_CONFIG", krb5ConfPath)
+    val ret = kinitProc.start().waitFor()
+    assert(ret === 0, "kinit failed")
+
+    val conn = DriverManager.getConnection(jdbcUrlWithConf(kerberosTgtJdbcUrl), user, "")
+    try {
+      val statement = conn.createStatement()
+      val resultSet = statement.executeQuery("select engine_name()")
+      assert(resultSet.next())
+      assert(resultSet.getString(1).nonEmpty)
+    } finally {
+      conn.close()
+    }
+  }
```

Comment:
Better to call `kdestroy` to avoid potential effects to other tests

Reply:
Thanks, added